vendor/k8s.io/kubernetes/staging/src/k8s.io/client-go/tools/clientcmd/client_config.go

```diff
@@ -100,6 +100,18 @@ func NewInteractiveClientConfig(config clientcmdapi.Config, contextName string,
 	return &DirectClientConfig{config, contextName, overrides, fallbackReader, configAccess, promptedCredentials{}}
 }
 
+func NewClientConfig(config clientcmdapi.Config) ClientConfig {
```

Comment:
How were you able to pass commit validation without an UPSTREAM commit?

Reply:
> How were you able to pass commit validation without an UPSTREAM commit?

uh, not sure. I'll update the pull though.